src/robot/libdocpkg/xmlwriter.py

```diff
@@ -13,8 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-from datetime import datetime
-
+from robot.libdocpkg.output import get_timestamp_for_doc
 from robot.utils import XmlWriter
```

Comment:
We've used relative imports with package internal imports. That would change these imports to:

```python
from robot.utils import XmlWritre

from .output import get_timestamp_for_doc
```

Reply:
Interesting, I don't remember seeing that recommendation in PEP-8 earlier. Anyway, our style has been doing relative inter-package imports and only use absolute imports when importing something from other top-level packages. I think it makes sense with packages with lot of sub-packages in general, but most importantly imports should use consistent style.